lib/figures/bishop.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative 'figure'
+
+class Bishop
+  attr_reader :row, :column, :possible_moves, :color, :display
+
+  include Figure
+
+  def initialize(color, row, column)
+    @color = color
+    @row = row
+    @column = column
+    @display = { white: '♝', black: '♗' }
+    @move_pattern = [[1, 1], [2, 2], [3, 3], [4, 4], [5, 5], [6, 6], [7, 7],
+                     [-1, -1], [-2, -2], [-3, -3], [-4, -4], [-5, -5], [-6, -6], [-7, -7],
+                     [-1, 1], [-2, 2], [-3, 3], [-4, 4], [-5, 5], [-6, 6], [-7, 7],
+                     [1, -1], [2, -2], [3, -3], [4, -4], [5, -5], [6, -6], [7, -7]]
+    @possible_moves = find_possible_moves
```

Comment:
Why assign `@possible_moves` to a variable, if you have to recalculate every time you change the position` ?
This seems like `@possible_moves` should be a method, no? And one implemented in the parent class, at that.